composer.json

```diff
@@ -13,7 +13,7 @@
         "php": ">=5.5",
         "ext-hash": "*",
         "ext-json": "*",
-        "ext-mongodb": "^1.5.0"
+        "ext-mongodb": "^1.5.2"
```

Comment:
This ensures that [CDRIVER-2744](https://jira.mongodb.org/browse/CDRIVER-2744) is in place via libmongoc 1.12.